tests/roles/manila_adoption/templates/manila_cephfs.yaml.j2

```diff
@@ -0,0 +1,58 @@
+spec:
+  manila:
+    enabled: true
+    apiOverride:
+      route: {}
+    template:
+      databaseInstance: openstack
+      databaseAccount: manila
+      manilaAPI:
+        customServiceConfig: |
+          [DEFAULT]
+          {% if manila_backend == "cephnfs" -%}
+          enabled_share_protocols=nfs
+          {% else -%}
+          enabled_share_protocols=cephfs
+          {%- endif %}
+
+        replicas: 1
+        networkAttachments:
+          - internalapi
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+              spec:
+                type: LoadBalancer
+      manilaScheduler:
+        replicas: 1
+      manilaShares:
+        share1:
+          customServiceConfig: |
+            [DEFAULT]
+            enabled_share_backends=cephfs
```

Comment:
@gouthampacha I assume it's ok to not have `enabled_share_protocols` in the `share` config.

Reply:
ack; makes sense..